Assets/auv_package/Scripts/CameraDataPublisher.cs

```diff
@@ -9,110 +9,137 @@
 public class CameraDataPublisher : MonoBehaviour
 {
     ROSConnection ros;
+    public UInt32 cameraFrameRate = 10; // hz
     public string frontTopicName = "front_simulation";
     public string bottomTopicName = "bottom_simulation";
```

Comment:
Change topics for sonia_simulation/front_raw and sonia_simulation/bottom_raw to fit with other topics

Reply:
Shoud we use proc_simulation as prefix and use the same topic as the real cam like proc_simulation/bottom_raw. It will match the simulation topic of proc_control